TheFourFifths/consus-client
# [TFF-123] Printing barcode labels

### Summary

This replaces the less-than-ideal one-at-a-time QR printing page with a page for printing barcodes. It allows you to print multiple barcodes at a time, and they are printed by our QL-500 printer with small address labels.

### Checklist


### How to Review

1. Run the client, with the server on `dev`
2. Go the the view all models page, click the QR/barcode icon on a model
3. Verify that the model address was added to the list
4. Back out, and go to the view all items page
5. Add an item by clicking the QR/barcode icon
6. Click 'Back'
7. Add another item
8. Click "print" and save the PDF
9. Open the PDF
10. The labels should be printable as 1 1/7" x 2 3/7"

### Links

- [TFF-123](https://msoese.atlassian.net/browse/TFF-123)

### Images

![image](https://cloud.githubusercontent.com/assets/5055041/25568977/0e45381c-2dd3-11e7-9050-86bbc9bb7977.png)
